public/js/webcam.js

```diff
@@ -1,5 +1,7 @@
 let video = document.getElementById('video');
 let canvas = document.getElementById('canvas');
+let beforeCaptureGroup = document.getElementById('beforeCaptureGroup');
```

Comment:
Can we rename to beforeCaptureButtons for clarity?

Reply:
Oh yup sure